picard/ui/options/metadata.py

```diff
@@ -156,9 +173,78 @@ def set_nat_name_default(self):
     def set_enabled_states(self):
         translate_checked = self.ui.translate_artist_names.isChecked()
         translate_exception_checked = self.ui.translate_artist_names_script_exception.isChecked()
-        self.ui.artist_locale.setEnabled(translate_checked)
+        self.ui.select_locales.setEnabled(translate_checked)
+        self.ui.selected_locales.setEnabled(translate_checked)
         self.ui.translate_artist_names_script_exception.setEnabled(translate_checked)
-        self.ui.artist_script_exception.setEnabled(translate_checked and translate_exception_checked)
+        self.ui.ignore_script_frame.setEnabled(translate_checked and translate_exception_checked)
+
+    def open_locale_selector(self):
+        dialog = MultiLocaleSelector(self)
+        dialog.show()
+
+
+class MultiLocaleSelector(PicardDialog):
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.ui = Ui_MultiLocaleSelector()
+        self.ui.setupUi(self)
+        self.ui.button_box.accepted.connect(self.save_changes)
+        self.ui.button_box.rejected.connect(self.reject)
+        self.move_view = MoveableListView(self.ui.selected_locales, self.ui.move_up, self.ui.move_down)
+        self.ui.add_locale.clicked.connect(self.add_locale)
+        self.ui.remove_locale.clicked.connect(self.remove_locale)
+        self.ui.selected_locales.currentRowChanged.connect(self.set_button_state)
+        self.load()
+
+    def load(self):
+        self.ui.selected_locales.clear()
+        for locale in self.parent().current_locales:
+            label = ("    " if '_' in locale else "") + ALIAS_LOCALES[locale]
```

Comment:
Locale should be translated.

I also see another issue with those indentations: this doesn't work for Right to Left languages (spacing should be on the other side).

So we should prolly have a translatable format string + translated locale name:

```python
        def indented_translated_locale(locale, level):
            return _("{indent}{locale}").format(
                indent="    " * level,
                locale=_(ALIAS_LOCALES[locale])
            )

        for locale in self.parent().current_locales:
            level = 1 if '_' in locale else 0
            label = indented_translated_locale(locale, level)
            item = QtWidgets.QListWidgetItem(label)
            item.setData(QtCore.Qt.UserRole, locale)
            self.ui.selected_locales.addItem(item)
        self.ui.selected_locales.setCurrentRow(0)

        self.ui.available_locales.clear()
        for (locale, name, level) in iter_sorted_locales(ALIAS_LOCALES):
            label = indented_translated_locale(locale, level)
            item = QtWidgets.QListWidgetItem(label)
            item.setData(QtCore.Qt.UserRole, locale)
            self.ui.available_locales.addItem(item)
        self.ui.available_locales.setCurrentRow(0)
```

Note, in the loop above, `name` has no use, perhaps we should also modify `iter_sorted_locales()`

Reply:
Yeah, I completely forgot about the translations for the UI lists and such.  I'll have a look at that.